src/Interpolators.f90

```diff
@@ -0,0 +1,146 @@
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!                          Futility Development Group                          !
+!                             All rights reserved.                             !
+!                                                                              !
+! Futility is a jointly-maintained, open-source project between the University !
+! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+! can be found in LICENSE.txt in the head directory of this repository.        !
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!> @brief A Fortran 2003 module implementing interpolation routines for input
+!>        arrays returning either interpolated values or a flat "extrapolation"
+!>        when interpolation is out of range.
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+MODULE Interpolators
```

Comment:
Technically the code standard says this should be `InterpolatorsModule`... admittedly, that rule was made assuming most module define classes, and appending `Module` was a way to avoid name collisions, but we should still get in the habit of following the standard